<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameStartWindow</class>
 <widget class="QWidget" name="GameStartWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>200</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>323</width>
    <height>181</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Начало игры</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="tabShape">
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabNewSession">
      <attribute name="title">
       <string>Новая сессия</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="1">
          <widget class="QLineEdit" name="editSessionKeyNew">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="labelNewSession">
           <property name="text">
            <string>ID сессии</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="btnNewSession">
         <property name="text">
          <string>Создать сессию</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabJoinToSession">
      <attribute name="title">
       <string>Существующая сессия</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QFormLayout" name="formLayout_2">
         <item row="0" column="1">
          <widget class="QLineEdit" name="editSessionKeyJoin"/>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="labelJoinSession">
           <property name="text">
            <string>ID сессии</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="btnJoin">
         <property name="text">
          <string>Подключиться</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
